src/DependencyInjection/Admin/AutoConfiguredODMAdminTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\DependencyInjection\Admin;
+
+trait AutoConfiguredORMAdminTrait
```

Comment:
```suggestion
trait AutoConfiguredODMAdminTrait
```

Also, this should be in the SonataDoctrineMongoDBBundle IMHO.

Reply:
Yes, indeed, i'll remove it